File: app/src/main/java/com/abdul/b2harmanandroid/Employee.java
package com.abdul.b2harmanandroid;

public class Employee {
    String eName;
    int eAge;
    boolean isElgible;

    public Employee(String eName, int eAge, boolean isElgible) {
        this.eName = eName;
        this.eAge = eAge;
        this.isElgible = isElgible;
    }

    public String geteName() {
        return eName;
    }

    public void seteName(String eName) {
        this.eName = eName;
    }

    public int geteAge() {
        return eAge;
    }

    public void seteAge(int eAge) {
        this.eAge = eAge;
    }

    public boolean isElgible() {
        return isElgible;
    }

    public void setElgible(boolean elgible) {
        isElgible = elgible;
    }
}

File: app/src/main/java/com/abdul/b2harmanandroid/Employee.kt
package com.abdul.b2harmanandroid

class Employee(var eName: String, var eAge: Int, var isElgible: Boolean) {
    fun geteName(): String {
        return eName
    }

    fun seteName(eName: String) {

              this.eName = eName
    }

    fun geteAge(): Int {
        return eAge
    }

    fun seteAge(eAge: Int) {
        this.eAge = eAge
    }

    companion object {
        var COMPANY_NAME = "harman" //class memory
    }
}